ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/plugins/ContentNegotiationTest.kt

```diff
@@ -578,32 +578,6 @@ class ContentNegotiationTest {
         }
     }
 
-    @Test
```

Comment:
why this test is removed?

Reply:
The test was checking the generic `accept` header verifying, which is not responsibility of `ContentNegotiation` anymore